index.html

```diff
@@ -0,0 +1,130 @@
+			</div>
+			<div class="description">Gumbo beet greens corn soko endive gumbo gourd. Parsley shallot courgette tatsoi pea sprouts fava bean collard greens dandelion okra wakame tomato. Dandelion cucumber earthnut pea peanut soko zucchini.</div>
+			<div class="availability">Available</div>
+		</section>
+<!-- card cection 2 -->
+		<section class="product-specs">
+			<h2>Specifications</h2>
+			<p class="size">Size: 1" x 1.5"</p>
+			<p class="weight">Weight: 10-60 lbs.</p>
+			<footer>
+			 <p class="dates-valid">Specifications valid through Dec. 31, 2016.</p>
+			</footer>
+		</section>
+<!-- card section 3 -->
+		<section class="product-pricing">
+			<h2>Pricing</h2>
+			<p class="prices">1: $22</p>
+			<p class="prices">5: $100</p>
+			<p class="prices">10-pack: $150</p>
+		</section>
+	</article>
+
+	<article class="product-card">
+		<header class="product-name">
+			<h1>Pop-up Office</h1>
+		</header>
+<!-- card section 1 -->
+		<section class="product">
+			<div class="image">
+				<img src="product-images/pop-up-office.jpg" alt="pop-up-office">
+			</div>
+			<div class="description">Gumbo beet greens corn soko endive gumbo gourd. Parsley shallot courgette tatsoi pea sprouts fava bean collard greens dandelion okra wakame tomato. Dandelion cucumber earthnut pea peanut soko zucchini.</div>
+			<div class="availability">Available</div>
+		</section>
+<!-- card section 2 -->
+		<section class="product-specs">
+			<h2>Specifications</h2>
+			<p class="size">Size: 1" x 1.5"</p>
+			<p class="weight">Weight: 10-60 lbs.</p>
+			<footer>
+			 <p class="dates-valid">Specifications valid through Dec. 31, 2016.</p>
+			</footer>
+		</section>
+<!-- card section 3 -->
+		<section class="product-pricing">
+			<h2>Pricing</h2>
+			<p class="prices">1: $22</p>
+			<p class="prices">5: $100</p>
+			<p class="prices">10-pack: $150</p>
+		</section>
+	</article>
+
+		<article class="product-card">
+		<header class="product-name">
+			<h1>Silent Boombox</h1>
+		</header>
+<!-- card section 1		 -->
+		<section class="product">
+			<div class="image">
+				<img src="product-images/boom.jpg" alt="Bad Memories">
+			</div>
+			<div class="description">Gumbo beet greens corn soko endive gumbo gourd. Parsley shallot courgette tatsoi pea sprouts fava bean collard greens dandelion okra wakame tomato. Dandelion cucumber earthnut pea peanut soko zucchini.</div>
+			<div class="availability">Available</div>
+		</section>
+<!-- card section 2 -->
+		<section class="product-specs">
+			<h2>Specifications</h2>
+			<p class="size">Size: 1" x 1.5"</p>
+			<p class="weight">Weight: 10-60 lbs.</p>
+			<footer>
+			 <p class="dates-valid">Specifications valid through Dec. 31, 2016.</p>
+			</footer>
+		</section>
+<!-- card section 3 -->
+		<section class="product-pricing">
+			<h2>Pricing</h2>
+			<p class="prices">1: $22</p>
+			<p class="prices">5: $100</p>
+			<p class="prices">10-pack: $150</p>
+		</section>
+	</article>
+</body>
+</html>
```

Comment:
Always include a new line at the end of every file. Consider adding ensure_newline_at_eof_on_save to your sublime preferences: http://docs.sublimetext.info/en/latest/reference/settings.html#automatic-behavior